pkgs/applications/audio/parlatype/default.nix

```diff
@@ -53,6 +54,7 @@ stdenv.mkDerivation rec {
     gtk4
     hicolor-icon-theme
     isocodes
+    libadwaita
```

Comment:
This should be in `buildInputs`.

Reply:
It *is* in buildInputs though? :)

Reply:
Oh, sorry, I didn’t see that the diff was folded in the middle.